models/scheduler.go

```diff
@@ -356,7 +356,7 @@ func ListSchedulerReleases(db interfaces.DB, schedulerName string) (
 ) {
 	_, err = db.Query(
 		&versions,
-		"SELECT version, created_at FROM scheduler_versions WHERE name = ?",
+		"SELECT version, created_at FROM scheduler_versions WHERE name = ? ORDER BY version",
```

Comment:
The order by should be `ORDER BY created_at ASC` because version is a string

Reply:
we can order by strings as well

Reply:
It wouldn't work for this case:
```
v10.0
v100.0
v11.0

Reply:
you're right :(

Reply:
Agreed, will fix this